Source/Core/DolphinQt/Achievements/AchievementLeaderboardWidget.cpp

```diff
@@ -87,7 +87,7 @@ void AchievementLeaderboardWidget::UpdateData(bool clean_all)
   }
   for (auto row : m_leaderboard_order)
   {
-    UpdateRow(row.second);
+    UpdateRow(row.first);
```

Comment:
Btw I didn't put this together until now but I think this is what iwubcode was talking about here: https://github.com/dolphin-emu/dolphin/pull/12896#discussion_r1656486471

Reply:
Yeah and then when Lily questioned it, I lost it after a long day 😄 .  [This line](https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/DolphinQt/Achievements/AchievementLeaderboardWidget.cpp#L59), would ultimately pass to UpdateRow, which could pass down to [FetchBoardInfo](https://github.com/dolphin-emu/dolphin/blob/ca40eeecff28c68cea7e195ec07fa0664ac3a36d/Source/Core/Core/AchievementManager.cpp#L774) which would call into rc.  But this change will fix it.

Not critical but maybe we could rename `UpdateRow` to `UpdateLeaderboard`?